tests/foreman/cli/test_lifecycleenvironment.py

```diff
@@ -122,8 +120,6 @@ def test_positive_create_with_description(self):
         :expectedresults: Lifecycle environment is created with Library as
             prior
 
-
-        :CaseImportance: Critical
```

Comment:
Changing importance from Critical to High changes tier assignment.

Reply:
moved to tier2